src/main/java/gregtech/api/metatileentity/multiblock/IControlRodPort.java

```diff
@@ -0,0 +1,9 @@
+package gregtech.api.metatileentity.multiblock;
+
+public interface IControlRodPort {
```

Comment:
Are there plans for more control rod port types? Right now I don't really see the need for this interface. The `hasModeratorTip()` call is only used in one place, and instead of checking if the MTE is an instance of `IControlRodPort`, you could just as easily check if it is an instance of `MTEControlRodPort`. So this interface is a little unneeded, at least in my opinion.